extensions/lifecycle/tasks/LifecycleObjectTask.js

```diff
@@ -0,0 +1,95 @@
+const async = require('async');
+const AWS = require('aws-sdk');
+
+const BackbeatTask = require('../../../lib/tasks/BackbeatTask');
+const AccountCredentials =
+    require('../../../lib/credentials/AccountCredentials');
+
+const attachReqUids =
+    require('../../replication/utils/attachReqUids');
+
+
+class LifecycleObjectTask extends BackbeatTask {
+    /**
+     * Process a lifecycle object entry
+     *
+     * @constructor
+     * @param {QueueProcessor} qp - queue processor instance
+     */
+    constructor(qp) {
+        const qpState = qp.getStateVars();
+        super({
+            retryTimeoutS: qpState.repConfig.queueProcessor.retryTimeoutS,
```

Comment:
Might be separating this to its own process, so use instead `lifecycle.retryTimeoutS` depending on how it is defined in `qpState`